src/olympia/addons/models.py

```diff
@@ -1555,37 +1559,114 @@ def _is_recommended_theme(self):
             ).exists()
         )
 
-    def promoted_group(self, *, currently_approved=True):
+    def promoted_groups(self, *, currently_approved=True):
         """Is the addon currently promoted for the current applications?
 
-        Returns the group constant, or NOT_PROMOTED (which is falsey)
-        otherwise.
+        Returns the list of group constants.
 
         `currently_approved=True` means only returns True if
         self.current_version is approved for the current promotion & apps.
         If currently_approved=False then promotions where there isn't approval
         are returned too.
         """
-        from olympia.promoted.models import PromotedAddon
 
         try:
-            promoted = self.promotedaddon
-        except PromotedAddon.DoesNotExist:
-            return NOT_PROMOTED
-        is_promoted = not currently_approved or promoted.approved_applications
-        return promoted.group if is_promoted else NOT_PROMOTED
+            promoted_addons = self.promoted_addons.all()
+        except ValueError:
```

Comment:
Why would we expect a value error here? Can you add a comment for why that would happen?